include/inventorymenu.h

```diff
@@ -57,9 +57,12 @@ class InventoryMenu final : public ChildMenu
     void updateIdDisp();
     void initIdScreen();
     void exitIdScreen();
+    static bool getAllCharsPixls(MenuButton * button, void * param);
+    const char * mAllCharsPixlsMsg;
```

Comment:
The class fields are grouped by what screen they're used on, these fields would go in the slot selection screen section